pkg/template/read.go

```diff
@@ -108,3 +109,16 @@ func templatesIterator(templatesPath string) (func() (Spec, bool, error), func()
 		}
 	}, cancel
 }
+
+func fillDefaultSpecValues(s *Spec) {
+	for i := range s.Parameters {
```

Comment:
you can get index, value from range here:
for i, v := range s.Parameters{ v.Type....

Reply:
ok, cannot you just use a pointer? *s.Parameters?

Reply:
The only option I see is to make s.Parameters array of pointers instead of array of values. But I don't think that it's that necessary.
It's just the style of for loop